src/witan/models/run_models.clj

```diff
@@ -127,11 +129,30 @@
   (core/looping-test (get-datasets inputs gss-code)
                      params))
 
-(defn output-ccm
+(def cli-options
+  [["-i" "--input-config FILEPATH" "Filepath for the config file with inputs info"
+    :default "resources/default_config.edn"]
+   ["-o" "--output-projections FILEPATH" "Filepath for the output projections"
+    :default "resources/ccm_projections.csv"]
```

Comment:
Probably not a good idea to provide a default output, especially one with a directory that mightn't exist